deploy/charts/cert-manager/values.yaml

```diff
@@ -70,7 +70,7 @@ podDisruptionBudget:
   # or a percentage value (e.g. 25%)
 
 # Comma separated list of feature gates that should be enabled on the
-# controller pod & webhook pod.
+# controller pod.
 featureGates: ""
```

Comment:
Can we change the type of this field to a map? That is an improvement over the current string & it will notify our users that something has to be changed.

Reply:
We can build the string by using {{ join ","  ... }}

Reply:
> it will notify our users that something has to be changed

what do you mean by this?

Reply:
I think it'd be preferable that it was the same as the same one for controller which is already string and probably shouldn't be changed